docs/modular-resources/create/_index.md

```diff
@@ -268,14 +268,14 @@ import (
     "fmt"
     "math"
 
-    "github.com/edaniels/golog"
     "github.com/golang/geo/r3"
     "github.com/pkg/errors"
     "go.uber.org/multierr"
 
     "go.viam.com/rdk/components/base"
     "go.viam.com/rdk/components/base/kinematicbase"
     "go.viam.com/rdk/components/motor"
+    "go.viam.com/rdk/logging"
```

Comment:
I see 7 remaining references to `golog` in this file. All should be changed to `logging`. Lines:
* 301
* 376
* 521
* 526
* 656
* 664
* 669